runtime/rococo/src/lib.rs

```diff
@@ -110,7 +110,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v1.5"),
 	authoring_version: 0,
-	spec_version: 231,
+	spec_version: 232,
```

Comment:
```suggestion
	spec_version: 231,
```

Reply:
Should I bump impl version instead? Or just not bump anything?

Reply:
Nothing changed, nothing needs to be bumped